lib/express_validator.js

```diff
@@ -360,6 +366,13 @@ function validateSchema(schema, req, loc, options) {
         continue;
       }
 
+      if (methodName === 'optional') {
+        /**
+         * We already done this before
```

Comment:
Change this comment to something like: "We explicitly do optional first if it is present, so skip it here."